### DISCUSSION TITLE
Container getting CrashLoopBackOff

### DISCUSSION THREAD

**Discussion by Ros-web** [egovernments/Digit-Core#263] [Bugs]
Hi , I have installed Digit-Core Application on aws eks , application also accessible over broweser , but some of container getting  CrashLoopBackOff here some screenshoot,
![image](https://github.com/egovernments/Digit-Core/assets/60926594/b0bf9c21-9b34-4b7a-b85b-5a4350548e90)

**Comment by nikhilmulinti** [comment]
Please share the output of kubectl describe <pod_name> -n egov
